tools/nl/embeddings/input/base/convert.py

```diff
@@ -0,0 +1,38 @@
+import csv
+from pathlib import Path
+from typing import Dict, List
+
+from absl import app
+from absl import flags
+from ruamel.yaml import YAML
+
+_COL_DCID = 'dcid'
+_COL_SENTENCE = 'sentence'
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string('input_file', '', 'The input csv file path')
+
+yaml = YAML()
+yaml.indent(mapping=3, sequence=2,
+            offset=0)  # sequence and offset have their default values here
+yaml.preserve_quotes = True
+yaml.width = 800  # this is the output line width after which wrapping occurs
+
+
+def main(_):
+  file_name = FLAGS.input_file
+  sv2texts: Dict[str, List[str]] = {}
+
+  with open(file_name) as f:
```

Comment:
nit: should we support vice versa too?

Reply:
This is just to convert current input csv to input yaml. Do you mean to generate the preindex as yaml too?

Reply:
I was thinking in case we need to curate in sheets, but never mind, we probably won't go the other way, but mostly only csv -> yaml.

Reply:
yea, i plan to update the google sheets tool to read yaml later.